python/lsst/ip/diffim/imageMapReduce.py

```diff
@@ -275,15 +276,36 @@ def run(self, mapperResults, exposure, **kwargs):
                     wtsView = afwImage.ImageI(weights, item.getBBox())
                     wtsView.getArray()[isNotNan] += 1
 
+        isNan = (np.isnan(newMI.getImage().getArray()) |
+                  np.isnan(newMI.getVariance().getArray()))
+        if np.sum(isNan) > 0:
+            # set mask to INVALID for pixels where produced exposure is NaN
+            mask = newMI.getMask()
+            # New mask plane - for debugging map-reduced images
+            mask.addMaskPlane('INVALID_MAPREDUCE')
+            bad = mask.getPlaneBitMask(['INVALID_MAPREDUCE', 'BAD', 'NO_DATA'])
```

Comment:
These mask planes should not be hard coded in the routine, but configurable from the config class with these as defaults. In the future there may be some reason to set more.